apps/frontend/src/widget/MusicPlayer.tsx

```diff
@@ -69,15 +69,17 @@ export const MusicPlayer = () => {
   /*  TODO: Add cover to backend */
```

Comment:
We're missing the issue number here.

Reply:
Previously, we didn't use GitHub Project, so we removed the comment. Now, standard convention: use Issues and GitHub Project, and note the issue number in the comment.